include/vcpkg/base/messages.h

```diff
@@ -318,25 +321,19 @@ namespace vcpkg::msg
     DECLARE_MESSAGE(BothYesAndNoOptionSpecifiedError,
                     (msg::option),
                     "",
-                    "error: cannot specify both --no-{option} and --{option}.");
+                    "cannot specify both --no-{option} and --{option}.");
 
-    inline void print_warning(const LocalizedString& s)
-    {
-        print(Color::warning, format(msgWarningMessage).append(s).append_raw('\n'));
-    }
+    void print_warning(const LocalizedString& s);
```

Comment:
```suggestion
    void println_warning(const LocalizedString& s);
```

I find it very surprising that a function without `ln` in the name adds a newline at the end of my message.

Reply:
Done!